bodhi/templates/update.html

```diff
@@ -535,6 +535,8 @@
             % endif
           % elif update.meets_testing_requirements and getattr(update.request, 'description', None) != 'stable':
             <a id='stable' class="btn btn-sm btn-success"><span class="glyphicon glyphicon-circle-arrow-right"></span> Push to Stable</a>
+          % elif update.stable_karma not in (0, None) and update.karma >= update.stable_karma and not update.autokarma and getattr(update.request, 'description', None) != 'stable':
```

Comment:
Why `and not update.autokarma`? This means that the manual push is still not possible if autokarma is enabled, but did not trigger for some reason (e.g., because it was enabled only after the karma value was reached). I don't see any valid reason for that restriction.

This and the other issue I pointed out in this review make the feature from pull request #821 entirely non-functional.